cyprecice/cyprecice.pyx

```diff
@@ -1193,6 +1193,81 @@ cdef class Interface:
         self.thisptr.readScalarData (data_id, vertex_id, _value)
         return _value
 
+    def set_mesh_access_region (self, mesh_id, bounding_box):
+        """
+        EXPERIMENTAL
```

Comment:
SInce we used the numpy documentation as a template, you can also check here: https://github.com/numpy/numpy/blob/2d112a98ed7597c4120b31908384ae09b0304659/numpy/array_api/__init__.py#L1-L125

Some observations:

* experimental state is only briefly mentioned in the docs.
* a warning is raised.